Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:lts-alpine
+
+RUN mkdir -p /usr/src/app
+
+WORKDIR /usr/src/app
+
+COPY . . 
+
+RUN npm run build
+
+RUN npm install redis@0.8.1 && \
+    npm install pg@4.1.1 && \
+    npm install memcached@2.2.2 && \
+    npm install aws-sdk@2.738.0 && \
+    npm install rethinkdbdash@2.3.31
+
+ENV HOST 0.0.0.0
```

Comment:
Shouldn't this be defaulted to `127.0.0.1` to keep consistency with config? I prefer localhost over this solution to not open our app to whole internet and manage its access via proxying port 7777 with stuff like nginx.

Reply:
This would be controlled by the ports you set with the docker commands. By default no ports are exposed. You can still only open to localhost by binding `127.0.0.1:7777` or to all interfaces with `7777:7777`.

If this is `HOST 127.0.0.1` then you can only connect to it from inside the container.